version: "3.8"
services:

  db_main:
    container_name: pycryptobot_db_main
    image: postgres:12
    environment:
      POSTGRES_DB: pycryptobot
      POSTGRES_USER: pycryptobot
      POSTGRES_PASSWORD: pycryptobot
    ports:
      - "5432:5432"
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U pycryptobot"]
      interval: 5s
      timeout: 2s
      retries: 10

  db_test:
    container_name: pycryptobot_db_test
    image: postgres:12
    environment:
      POSTGRES_DB: pycryptobot_test
      POSTGRES_USER: pycryptobot_test
      POSTGRES_PASSWORD: pycryptobot_test
    ports:
      - "5433:5432"
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U pycryptobot_test"]
      interval: 5s
      timeout: 2s
      retries: 10
